includes/public/layouts/rating-tpl.php

```diff
@@ -17,8 +17,9 @@
 	$is_disabled = apply_filters( 'wppr_disable_price_richsnippet', false );
 	$currency = preg_replace( '/[0-9.,]/', '', $review['price'] );
 	if ( ! $is_disabled ) {
+		$country_iso    = apply_filters( 'wppr_currency_code', $currency );
 		$sub_title_info = '<span itemprop="offers" itemscope itemtype="http://schema.org/Offer">
-                                <span itemprop="priceCurrency">' . $currency . '</span>
+                                <span itemprop="priceCurrency" content="' . $country_iso . '">' . $currency . '</span>
```

Comment:
@contactashish13 there is a misunderstanding here, this code is redundant and i forgot to change it. We use json-ld for schema.org generation. there is a template file called json-ld which you can alter to add this.

Reply:
@selul have added there as well. it calls get_currency() method where I've called the filter